Marble Blast Platinum/platinum/server/scripts/pads.cs

```diff
@@ -91,11 +91,12 @@
 }
 
 function EndPad_MBM::onAdd(%this,%obj) {
-   if ($pref::spchanges && %obj.isTemperable $= "1") {
+   if (($pref::spchanges && %obj.isTemperable $= "1") || ($Server::MissionFile $= ($usermods @ "/data/missions/templates/UltraTemplate.mis") && %obj.tochange == 1)) {   //Because the Ultra Template level needs to have the MBM End Pad (it will crash when making a new level with the MBU Pad), there is a simple mechanism in place to switch out the MBM pad in that level with an MBU pad. ~Connie
```

Comment:
Is this needed in combination with the lines above?

https://github.com/The-New-Platinum-Team/PlatinumQuest-Dev/pull/120/files#diff-dfd5f35a2e895abdddac8a3b4c719b155af9490871c50108b22174c17b7a0294R114-R116

They seem to do the same thing, so I'm wondering if one is needed without the other.

Reply:
One handles the pad when the level gets created with the Ultra template (latter one), one handles the pad when you select the Ultra template in the Level Select Screen (first one).
I tried to make it so there weren't two things which do this, but the game ended up crashing when I tried to make a level with the Ultra Template.
Idk if I'm dumb and there's some way to do this, but yeah

Reply:
Ok yeah I believe it. I tried a few things and couldn't quite get it to work. Would be nice to figure out what's causing the crash in the first place but this works for now